### PR TITLE
Fixes for Vendor create/edit forms

### PR DESCRIPTION
**Pull request by BradHN1**
Fixes issue 249
Also change some defaults so they don't say "wayland" in silly places

### CODE REVIEW

File: v2.0/app/api/ui/superAdminMenu.js [hunk 1]
```diff
@@ -236,5 +214,27 @@ module.exports = [
...
+    name: 'Tag Collections',
+    icon: 'md-reorder',
+    child: [
+      {
+        key: 'about-categories',
+        name: 'Tags & Tag Collections',
+        title: true
+      },
+      {
+        key: 'add-vendor',
```
Comment (massenergize-admin) [inline v2.0/app/api/ui/superAdminMenu.js:229]: are we sure we want to put add-vendor as the key here?  maybe add-category?

File: v2.0/app/config/config.json [hunk 2]
```diff
@@ -1,5 +1,5 @@
 {
-  "IS_LOCAL": false,
+  "IS_LOCAL": true,
```
Comment (massenergize-admin) [inline v2.0/app/config/config.json:2]: we dont want to checkin is local= true right?

File: v2.0/app/containers/MassEnergizeSuperAdmin/Events/CreateNewEventForm.js [hunk 3]
```diff
@@ -240,7 +240,7 @@ class CreateNewEventForm extends Component {
               {
                 name: 'unit',
                 label: 'Unit Number',
-                placeholder: 'eg. wayland',
+                placeholder: 'eg. "2A"',
```
Comment (massenergize-admin) [inline v2.0/app/containers/MassEnergizeSuperAdmin/Events/CreateNewEventForm.js:243]: nice

File: v2.0/app/api/ui/superAdminMenu.js (hunk 1 above)
Comment (BradHN1) [reply v2.0/app/api/ui/superAdminMenu.js:229]: Your right.  I've fixed that.  (Not my bug:)  I don't know what that key is used for really, but maybe it needs to be unique.
Is the file menu.js (which also had the bug) obsolete?  I assume so and will delete it.

File: v2.0/app/config/config.json (hunk 2 above)
Comment (BradHN1) [reply v2.0/app/config/config.json:2]: Right you are.  Fixed.

**Review by Opoku-Agyemang — APPROVED** [review]
LGTM ✅

**Pull request merged**